src/components/organisms/AdminVenueView/components/Spaces/Spaces.tsx

```diff
@@ -156,6 +156,7 @@ export const Spaces: React.FC<SpacesProps> = ({
                 isLoadingBackgrounds={isLoadingBackgrounds}
                 mapBackgrounds={mapBackgrounds}
                 venueName={venue.name}
+                spaceSlug={venue.slug}
```

Comment:
This one is pointless. 
The `slug` gets passed down to `createFirestoreVenueInput_v2()` but that one isn't using it, but still doing 
```ts
    const slug = createSlug(input.name);
```

and returning 
```ts
    name: slug,
    slug,
```

Reply:
This is a tricky one and I don't have the time to refactor it or decide which approach is the correct one.

It is in fact currently ignored and not used inside, but it will be in the near future. Right now `slug` is required field because of the typing, and it's passed but ignored. The current `createSlug` will always generate a new one when we change the name of the venue but that shouldn't be happening. The slug should have a separate field, which is already in the IA design.

Reply:
> It is in fact currently ignored and not used inside, but it will be in the near future.

☝️ then a `@debt` comment should suffice